DeliBuddy/app/src/main/java/yapp/android1/delibuddy/ui/address/AddressSharedViewModel.kt

```diff
@@ -29,10 +29,16 @@ sealed class AddressSharedEvent : Event {
     class CoordToAddress(val lat: Double, val lng: Double) : AddressSharedEvent()
 }
 
+sealed class SaveAddressEvent : Event {
+    object Idle : SaveAddressEvent()
+    class Success(val userAddress: String) : SaveAddressEvent()
+    object Failed : SaveAddressEvent()
+}
```

Comment:
이런식으로 이벤트 추가해봤는데 맞을까유...? 의견이 필용합니다... @LuvJoy

Reply:
AddressSharedEvent
```suggestion
sealed class AddressSharedEvent : Event {
    sealed class SaveAddressEvent : AddressSharedEvent {
        object Idle : SaveAddressEvent()
        class Success(val userAddress: String) : SaveAddressEvent()
        object Failed : SaveAddressEvent()
    }
}
```
sealed class 안에 sealed class를 넣어서 하는건 어떨까요?
그래도 하나로 묶는게 좋지 않을까 싶네요!